rubin_jupyter_utils/hub/scanrepo/parse_args.py

```diff
@@ -67,6 +67,12 @@ def parse_args(
             help="repository owner [{}]".format(lro),
             default=lro,
         )
+        parser.add_argument("--username",
```

Comment:
Can we call this --repo-username and --repo-password?  Or is that clear from context?

Reply:
I think it's about as clear as it can be considering it's a very confusion situation.  Since we're also dealing with git repos everywhere, repo is also a confusing name.  And we're getting rid of this.